README.md

```diff
@@ -217,6 +217,10 @@ Your first task is to build a hand of 10 letters for the user. To do so, impleme
 
 #### Distribution of Letters
 
+letter_pool = ["a", "a", "a", "a", "b", "b', "c"]
+
+pool_index = random integer in the range(len(letter_pool))
```

Comment:
I'm not sure if these changes were intended to be committed. If it's example data for your own learning while you were coding, we should remove it as part of our refactor and clean up when finishing a project.

Reply:
Oh, no, no I think this was part of our initial planning session, whoops 😂